web/gds-admin-ui/src/components/Auth/SignWithGoogle.js

```diff
@@ -5,21 +5,18 @@ import React from 'react'
 function SignWithGoogle({ loginResponse, text, clientId }) {
 
     React.useEffect(() => {
-        window.onload = function () {
-            if (window.google) {
-                window.google.accounts.id.initialize({
-                    client_id: clientId,
-                    callback: loginResponse,
-                    text,
-                });
-                window.google.accounts.id.renderButton(
-                    document.getElementById("buttonDiv"),
-                    { theme: "filled_blue", size: "large", margin: "0 auto" }
-                );
-            }
+        if (window.google) {
+            window.google.accounts.id.initialize({
+                client_id: clientId,
+                callback: loginResponse,
+                text,
+            });
+            window.google.accounts.id.renderButton(
+                document.getElementById("buttonDiv"),
+                { theme: "filled_blue", size: "large", margin: "0 auto" }
+            );
```

Comment:
does the removal of the `onload` event allow the button to be rendered only after the `GET /v2/authenticate` has been completed?

Reply:
Yes, it does. I removed it because the google doesn't render correctly and I have to reload the page